chain/client_wrapper.sh

```diff
@@ -44,13 +44,13 @@
 #     - Needs the password file and the key-set. (see FILES)
 #
 # FILES
-#   The configuration folder for Parity takes place at /home/parity/.local/share/io.parity.ethereum.
+#   The configuration folder for used client takes place at /home/openethereum/.local/share/io.parity.ethereum.
```

Comment:
the end `io.parity.ethereum` did not change?

Reply:
That is what their documentation states, but it is lying. By testing, I see that it uses `io.parity.ethereum` if it exists, and `openethereum` otherwise.
https://openethereum.github.io/wiki/Configuring-Parity-Ethereum

Reply:
okay, so thats sounds like it is only the fallback, so maybe we should switch to openethereum there too?

Reply:
Yes.

Reply:
I fixed it, the remaining places in the code where we have the word `parity` left are very few and easy to check, which should mean there is no mistake left. (config.toml under [parity], and the name of some rpc are still `parity_...`, the behaviour of which I checked locally with openethereum)

Reply:
sounds good